src/components/templates/AllReports.svelte

```diff
@@ -19,28 +18,46 @@
 <style lang="scss">
   // Importing fonts, vars, etc.
   @import 'src/styles/index.scss';
+
   * {
-    font-family: $standard-font;
+    font-family: 'Open Sans';
```

Comment:
I'll take a look at how to load the font-family in a more global way, but I'll come back to this later 😄 👍🏼